.gitignore

```diff
@@ -0,0 +1 @@
+.idea/*
```

Comment:
Can you remove this? Personal files (even though I have one too) need to be ignored in your global .gitignore file.